config.json

```diff
@@ -229,137 +229,152 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "f77dc7e3-35a8-4300-a7c8-2c1765e9644d",
-        "practices": [],
+        "practices": [
+          "basics"
+        ],
         "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
-          "strings"
-        ]
+        "difficulty": 1
       },
       {
         "slug": "two-fer",
         "name": "Two Fer",
         "uuid": "74515d45-565b-4be2-96c4-77e58efa9257",
-        "practices": [],
-        "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
-          "conditionals-if",
-          "strings"
-        ]
+        "practices": [
+          "strings",
+          "conditionals-if"
+        ],
+        "prerequisites": [
+          "basics",
+          "conditionals-if"
+        ],
+        "difficulty": 1
       },
       {
         "slug": "hamming",
         "name": "Hamming",
         "uuid": "ce899ca6-9cc7-47ba-b76f-1bbcf2630b76",
-        "practices": [],
-        "prerequisites": [],
-        "difficulty": 3,
-        "topics": [
-          "integers",
-          "loops",
-          "strings"
-        ]
+        "practices": [
+          "strings",
+          "for-loops"
+        ],
+        "prerequisites": [
+          "basics",
+          "strings",
```

Comment:
Concepts should not be in both practices and prerequisites

Reply:
In my interpretation, users could need the basics of a concept in order to solve an exercise and could end up practicing that same concept once the exercise is done.

Following your recommendation, I've changed `prerequisites` to include only the strict necessary concepts. The ones that can be practiced went to `practices`.